.golangci.yml

```diff
@@ -35,7 +35,7 @@ linters-settings:
   dupl:
     threshold: 150
   funlen:
-    Lines: 100
+    Lines: 105
```

Comment:
Can we apply the fix for a file and not change this for all files?

Reply:
ok, fixed